### DISCUSSION TITLE
Error in Fund Me Section

### DISCUSSION THREAD

**Discussion by Anshpatwal** [Cyfrin/foundry-full-course-cu#2453] [General]
![Capture](https://github.com/user-attachments/assets/3e4a2433-ad99-4ec6-893f-ad4836fcd89f)

Don't know why but i cant seem to deploy it with Meta Mask Injector on Remix IDE , I can deploy it with Remix Cancun but the Functions wont work somehow, I can't get the price of ETH , Need some help.

**Comment by EngrPips** [comment]
Hello @Anshpatwal, use the `wallet connect` option if the ` Meta Mask Injector` does not work for you.

**Comment by 0xArDANT** [comment]
It doesn't work on VM Cancun because Chainlink contracts aren't deployed here so you can't access them.
You should try again with Metamask or Wallet Connect; a wallet that can connect to tesnets such as Sepolia.

**Comment by EngrPips** [comment]
Interesting.

**Comment by 0xArDANT** [comment]
Did it work ?

**Comment by ko4ergins** [comment]
You should use answer instead of price
https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/shared/interfaces/AggregatorV3Interface.sol
![iScreen Shoter - Google Chrome - 241002224714](https://github.com/user-attachments/assets/c6a422ae-40b1-48ac-a8e6-e0c878a61dc4)
![iScreen Shoter - Google Chrome - 241002224341](https://github.com/user-attachments/assets/9e43de24-99ee-4c85-8035-89b699b1c1e7)
```
   function getPrice() internal view returns (uint256) {
        AggregatorV3Interface priceFeed = AggregatorV3Interface(
            0x694AA1769357215DE4FAC081bf1f309aDC325306
        );
        (, int256 answer, , , ) = priceFeed.latestRoundData();
        return uint256(answer * 10000000000);
    }
```

**Comment by EngrPips** [comment]
Interesting
